src/test/shared/utilities/timeoutUtils.test.ts

```diff
@@ -102,4 +102,61 @@ describe('timeoutUtils', async () => {
             longTimer.killTimer()
         })
     })
+
+    describe('waitUntil', async () => {
+        const testSettings = {callCounter: 0, callGoal: 0, functionDelay: 10}
```

Comment:
I like this pattern instead of declaring several variables. We should use this more often.